doc/rtd/topics/tutorials/qemu-debugging.rst

```diff
@@ -0,0 +1,46 @@
+.. _debug information:
```

Comment:
```suggestion
.. _qemu_debug_info:
```

Reply:
For consistency with all the other links I'd stick an underline in there. Also, debug information seems like it could have broad applications, so I'd also recommend making the tag more specific to the page (in case we need to use "debug information" for a more general page in the future)